io.github._7h0ma5.QLog.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github._7h0ma5.QLog
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qlog
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

Reply:
The wayland socket was intentionally removed because it caused issues. See my [previous comment](https://github.com/flathub/flathub/pull/1963#issuecomment-736732101).